packages/foundation/src/slots.tsx

```diff
@@ -40,13 +40,6 @@ export function withSlots<P>(
 ): ReturnType<React.FunctionComponent<P>> {
   const slotType = type as ISlot<P>;
   if (slotType.isSlot) {
```

Comment:
Hello, @KevinTCoughlin. I have small questions:
1. Can we take out `children = React.Children.toArray (children);` outside of check `if (slotType.isSlot) {}`? As I see this array is needed both `if` and for `else`.
2.  After we can add - **const** children = React.Children.toArray(children);